KonyAccelerate/controllers/mobile/frmFeedbackController/frmFeedbackController.js

```diff
@@ -0,0 +1,131 @@
+define({
+
+    //Type your controller code here 
```

Comment:
Remove the above line and spaces in the file

Reply:
Removed the commented text